TestAssets/TestProjects/StandaloneTestApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace StandaloneApp
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, World!");
```

Comment:
fixed.

---
In reply to: [66350842](https://github.com/dotnet/core-setup/pull/97#discussion_r66350842) [](ancestors = 66350842,66346432)